internal/storage/caching/fast_stat_bucket_test.go

```diff
@@ -783,3 +784,34 @@ func (t *StatObjectTest) TestRenameFolder() {
 	AssertEq(nil, err)
 	ExpectEq(result, folder)
 }
+
+func (t *StatObjectTest) TestShouldCallCreateFolder() {
```

Comment:
TEst method names are not clear. you are testing createFolder and mocking createFolder. Clearly mention which implemetation is throwig error or should be called.

Reply:
Can you please check now?